core/tests/performance_tests/lda_string_classifier_performance_test.py

```diff
@@ -130,7 +131,8 @@ def predict(self, num):
             raise Exception('No classifier found')
         classifier = (
             classifier_registry.Registry.get_classifier_by_algorithm_id(
-                feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput']))
+                feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput'][
```

Comment:
@anmolshkl . Well, this test is exclusively checking performance of LDA, so I don't think it might be a huge issue since this file will probably be deprecated. I'll do it in other places though.